where is the 'server.js' ?

I've install the package,and then there's not a file named server.js.How could I run the server locally ?

$ node server.js 8888
module.js:340
    throw err;
    ^

Error: Cannot find module 'D:\cc\jsconsole\jsconsole-master\server.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:136:18)
    at node.js:963:3

--- a/README.md
+++ b/README.md
@@ -20,13 +20,13 @@ This requires that you install [node.js](http://nodejs.org). Once installed, dow
 
 This will install the dependencies (in particular 1.8.x version of connect.js).
 
-Once installed, run (on port 80):
+Once installed, run (on port 8000):
 
-    node server.js
+    node .
 
 Or to run on a specific port (like 8080):
 
-    node server.js 8080
+    PORT=8080 node .
 
 Then check your own ip address of the machine it's running on (using `ipconfig` for windows or `ifconfig` for mac and linux). Then on the mobile phone, just visit that IP address and port you're running jsconsole on:
 

